ledger/ledger-api-errors/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -125,18 +126,19 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
       ) {
     case class Rejection(
         name: String,
+        metricNameLabel: String,
         queued: Long,
         limit: Int,
-        metricPrefix: String,
         fullMethodName: String,
     )(implicit errorLogger: ContextualizedErrorLogger)
         extends DamlErrorWithDefiniteAnswer(
-          s"The $name queue size ($queued) has exceeded the maximum ($limit). Api services metrics are available at $metricPrefix.",
+          s"The $name queue size ($queued) has exceeded the maximum ($limit). Metrics for queue size available at ${ExecutorServiceMetrics.CommonMetricsName.QueuedTasks}.",
```

Comment:
There seems to be two sources of metrics naming `ExecutorServiceMetrics` and `com.daml.metrics.Metrics`

Reply:
`ExecutorServiceMetrics` are specific to executors. We in fact create the `ExecutorServiceMetrics` instance in `com.daml.metrics.Metrics`[